app/javascript/styles/_search.scss

```diff
@@ -68,6 +68,20 @@
     font-weight: 500;
   }
 
+  #download-button.active {
```

Comment:
we now have the $action-color variable colors.scss for this color

Reply:
Addressed in b72b604.